lib/diagnostic.js

```diff
@@ -6,23 +6,26 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (words) {
+  words = [words.split(' ')]
```

Comment:
Part of the way there, next you need to make the words uppercase and remove any punctuation.